paymentsheet/src/main/java/com/stripe/android/link/LinkConfiguration.kt

```diff
@@ -41,6 +41,9 @@ internal data class LinkConfiguration(
     val customerIdForEceDefaultValues: String?
         get() = if (enableDisplayableDefaultValuesInEce) customerId else null
 
+    val enableLinkPaymentSelectionHint: Boolean
+        get() = flags["link_mobile_enable_payment_selection_hint"] == true
```

Comment:
perfect! let's make sure backend returns this as a `link_flag` and not a `link_setting`